quiche/src/recovery/bbr/per_ack.rs

```diff
@@ -270,7 +270,7 @@ fn bbr_enter_probe_bw(r: &mut Recovery, now: Instant) {
 
     bbr.state = BBRStateMachine::ProbeBW;
     bbr.pacing_gain = 1.0;
-    bbr.cwnd_gain = 2.0;
+    bbr.cwnd_gain = 1.25;
```

Comment:
This change diverges from original [BBR algorithm (draft-00)](https://datatracker.ietf.org/doc/html/draft-cardwell-iccrg-bbr-congestion-control-00#section-4.3.4.3). It will make cwnd growth less aggressive, but I believe it will take longer to find a max BDP in a variable high bandwidth/high rtt link. I think it would be great to discuss in bbr-dev@ mailing list.